packages/core/bootstrap/src/lib/external-adapter/validator.ts

```diff
@@ -18,121 +21,84 @@ import { Requester } from './requester'
 import { inputParameters } from './builder'
 
 export type OverrideType = 'overrides' | 'tokenOverrides' | 'includes'
+
 export class Validator {
-  input: any
-  customParams: any
+  input: {
+    id?: string
+    data?: any
+  }
+  inputConfigs: InputParameters
   options: Record<string, any[]>
-  validated: any
+  validated: {
+    id: string
+    data?: any
+    includes?: Override
+    overrides?: Override
+    tokenOverrides?: Override
+  }
   error: AdapterError | undefined
   errored: AdapterErrorResponse | undefined
+  shouldLogError: boolean
 
-  constructor(input = {}, customParams = {}, options = {}, shouldLogError = true) {
+  constructor(input = {}, inputConfigs = {}, options = {}, shouldLogError = true) {
     this.input = { ...input }
-    this.customParams = { ...customParams }
+    this.inputConfigs = { ...inputConfigs }
```

Comment:
Can this just be `this.inputConfigs = inputConfigs`?

Reply:
I'm guessing this is a catch-all to avoid changing the original objects passed into the validator (same with the other inputs). It doesn't look like `inputConfigs` is ever modified, so it isn't necessary, although it might still be good to keep as a safeguard for future changes. Thoughts?

Reply:
I see. Hmm in that case then I think it’s fine to keep it as it is.